gateway-provider-security-webappsec/src/main/java/org/apache/knox/gateway/webappsec/deploy/WebAppSecContributor.java

```diff
@@ -150,12 +165,24 @@ public void contributeFilter(DeploymentContext           context,
     }
   }
 
-  private void provisionConfig(ResourceDescriptor resource, Map<String,String> providerParams,
-      List<FilterParamDescriptor> params, String prefix) {
-    for(Entry<String, String> entry : providerParams.entrySet()) {
+  private void provisionConfig(ResourceDescriptor resource, Map<String, String> providerParams,
+                               List<FilterParamDescriptor> params, String prefix, boolean cutPrefix, boolean toLowerCase) {
```

Comment:
Why did you add these cure cutPrefix and toLowerCase - does the DOS filter treat the params as case sensitive?

Reply:
@lmccay - yes, Jetty's DOS filter is different than the other filters in this respect. As you can see from the samples Balazs gave us, the parameters in this filter:
- do not need any special prefix (such as `rate.limiting`)
- they are case sensitive